macros/GroupData.json

```diff
@@ -1834,10 +1857,15 @@
                             "RTCSessionDescriptionCallback",
                             "RTCStatsReport",
                             "RTCCertificate",
+                            "RTCIdentityProvider",
+                            "RTCIdentityAssertion",
+                            "RTCIdentityEvent",
+                            "RTCIdentityErrorEvent",
                             "RTCIdentityProviderRegistrar",
+                            "RTCTrackEvent",
```

Comment:
RTCTrackEvent: Yeah, not documented yet but it needs to be pretty urgently; I would like to do it in the next sprint or two.